pkg/sql/rowexec/columnbackfiller.go

```diff
@@ -107,5 +129,23 @@ func (cb *columnBackfiller) runChunk(
 		)
 		return err
 	})
+	if err == nil {
+		cb.commitWaitFns = append(cb.commitWaitFns, commitWaitFn)
+		if len(cb.commitWaitFns) == maxCommitWaitFns {
+			cb.runCommitWait(ctx)
+		}
+	}
 	return key, err
 }
+
+// runCommitWait consumes the commit-wait functions that the columnBackfiller
+// has accumulated across the chunks that it has backfilled. It calls each
+// commit-wait function to ensure that any dependent reads on the rows we just
+// backfilled observe the new column.
+func (cb *columnBackfiller) runCommitWait(ctx context.Context) {
+	for i, fn := range cb.commitWaitFns {
```

Comment:
should we veventf in here before the loop, if we anticipate we might hang in the loop for any length of time?